### PR TITLE
docs: improve documentation

### PR DESCRIPTION
**Pull request by dcoa**
## Description

This pull request enhances the documentation by:
- Updating tutor commands.
- Rewrite some paragraphs avoiding redundancy.  
- Fix the font family loading style

## Additional information

[JIRA](https://edunext.atlassian.net/browse/DS-971)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,9 +58,12 @@ source .tvm/bin/activate
 5. Run your project.
 
 ```bash
```
Comment (BryanttV) [inline README.md:60]: Please, add indentation to the code blocks
```suggestion
    ```bash
```

**Review by BryanttV — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dcoa) [reply README.md:60]: I'm using [fanced code blocks](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks), indentation is not needed.

Comment (BryanttV) [reply README.md:60]: Without Indentation
image
With Indentation
image

With indentation, the code block remains inside the numeral.

**Review by BryanttV — DISMISSED** [review]
LGTM, thanks!

File: README.md
Comment (BryanttV) [inline README.md:1]: Sorry for not having commented before but, could we add badges at the beginning of the README as it was done [here](https://github.com/eduNEXT/eox-hooks/pull/56)?

Comment (dcoa) [reply README.md:1]: Sure thing, I just forgot, thanks for reminding me of that.

**Review by mariajgrimaldi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
 # TVM
 
+![Maintainance Badge](https://img.shields.io/badge/Status-Maintained-brightgreen)
+![Test Badge](https://img.shields.io/github/actions/workflow/status/edunext/eox-tagging/.github%2Fworkflows%2Ftests.yml?label=Test)
```
Comment (BryanttV) [inline README.md:4]:
```suggestion
![Test Badge](https://img.shields.io/github/actions/workflow/status/edunext/tvm/.github%2Fworkflows%2Ftests.yml?label=Test)
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+![PyPI Badge](https://img.shields.io/pypi/v/eox-tagging?label=PyPI)
```
Comment (BryanttV) [inline README.md:5]: I think this badge can't be added :c

File: README.md (hunk 2 above)
Comment (dcoa) [reply README.md]: Sorry for that 😅

File: README.md (hunk 3 above)
Comment (dcoa) [reply README.md]: You right, I update it.

**Review by BryanttV — APPROVED** [review]

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**